test.py

```diff
@@ -923,7 +928,11 @@ def _prepare_pytest_params(self, options: argparse.Namespace):
             # https://docs.pytest.org/en/7.1.x/reference/exit-codes.html
             no_tests_selected_exit_code = 5
             self.valid_exit_codes = [0, no_tests_selected_exit_code]
-        self.args.append(str(self.suite.suite_path / (self.shortname + ".py")))
+
+        arg = str(self.suite.suite_path / (self.shortname + ".py"))
```

Comment:
A suggestion: for the benefit of pytest fanboys like myself, it would be nice to also support "name.py::case" and not just "name::case". I think it should be easy, just by not adding ".py" if there's already one?